jerry-main/main-unix-snapshot.c

```diff
@@ -306,8 +321,38 @@ process_generate (cli_state_t *cli_state_p, /**< cli state */
     return JERRY_STANDALONE_EXIT_CODE_FAIL;
   }
 
-  jerry_value_t snapshot_result;
+  if (is_import_literals)
+  {
+    uint8_t *sp_buffer_start_p = source_p + source_length + 1;
+    size_t sp_buffer_size = read_file (sp_buffer_start_p, literals_file_name_p);
+
+    if (sp_buffer_size > 0)
+    {
+      const char* sp_buffer_p = (const char*) sp_buffer_start_p;
+      uint32_t num_of_lit = 0;
 
+      do
+      {
+        char* sp_buffer_end_p = NULL;
+        jerry_length_t mstr_size = (jerry_length_t) strtol (sp_buffer_p, &sp_buffer_end_p, 10);
+        if (mstr_size > 0)
```

Comment:
Can it be 0?

Reply:
Shouldn't we detect that before calling strtol? What happens for invalid numbers?

Reply:
Do you mean something like this?
```
if ( '0' > *sp_buffer_p || *sp_buffer_p > '9')
{
  break;
}
```